generate_directory_indexes.py

```diff
@@ -93,20 +137,20 @@ def index_file_name(prefix, order_by, reverse_order):
 
 
 def format_date(d):
-    return datetime.datetime.fromtimestamp(
-        int(d)
-    ).strftime('%Y-%m-%d %H:%M:%S')
+    return datetime.fromtimestamp(int(d)).strftime('%Y-%m-%d %H:%M:%S')
+
 
-def format_size(num, suffix='B'):
+def format_size(a_number, suffix='B'):
     for unit in ['','Ki','Mi','Gi','Ti','Pi','Ei','Zi']:
-        if abs(num) < 1024.0:
-            return "%3.1f%s%s" % (num, unit, suffix)
-        num /= 1024.0
-    return "%.1f%s%s" % (num, 'Yi', suffix)
+        if abs(a_number) < 1024.0:
+            return "%3.1f%s%s" % (a_number, unit, suffix)
+        a_number /= 1024.0
+    return "%.1f%s%s" % (a_number, 'Yi', suffix)
```

Comment:
If possible we should be using `.format()` instead of `%` for formatting strings as `%` is deprecated.